Dockerfile

```diff
@@ -29,17 +31,8 @@ RUN mkdir -p /opt/android-sdk && cd /opt/android-sdk && \
     unzip *tools*linux*.zip && \
     rm *tools*linux*.zip
 
-# download and install Kotlin compiler
-ENV KOTLIN_VERSION 1.2.61
```

Comment:
So I'm guessing gradle will download Kotlin on demand?

Reply:
Yes, I'm pretty sure it is actually already doing that by itself and that version of the compiler is never used by the Android build => the Kotlin compiler is a dependency of the Kotlin gradle plugin
(maybe the KOTLIN_HOME variable overrides the normal behaviour but I doubt it)

And it's actually a lot better to not care about the version installed on our build machines and declare it project by project.

Reply:
👍